styles/pages/internship.scss

```diff
@@ -0,0 +1,88 @@
+.testimonial-item {
+  box-shadow: 0 0.125rem 0.5rem rgba(0, 0, 0, 0.3), 0 0.0625rem 0.125rem rgba(0, 0, 0, 0.2);
+  margin: 3px;
+  padding: 5px;
+}
+
+p {
+  font-size: 0.8em;
+  line-height: 150%;
+  margin: 0 22px 14px;
+
+  &.subheading {
+    font-size: 1em;
+  }
+}
```

Comment:
Can we avoid using element selectors - this is particularly pernicious since it's globally-scoped, and now import order will break specificity ties.